roles/openshift_master_certificates/tasks/main.yml

```diff
@@ -20,6 +20,8 @@
     - admin.kubeconfig
     - master.kubelet-client.crt
     - master.kubelet-client.key
+    - "{{ 'master.proxy-client.crt' if openshift.master.include_proxy_client_cert else omit }}"
+    - "{{ 'master.proxy-client.key' if openshift.master.include_proxy_client_cert else omit }}"
```

Comment:
Very right - it succeeds but only because `masters_needing_certs` is empty on my single node setup.